eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/OyHome.java

```diff
@@ -87,4 +89,25 @@ public Input get(final String name) throws FileNotFoundException {
         }
         return new InputOf(file);
     }
+
+    @Override
+    public boolean contains(final String name) {
+        boolean ret;
+        try {
+            final Path file = new Place(name).make(
+                this.home
+                    .resolve("pulled")
+                    .resolve(this.version),
+                "eo"
+            );
+            ret = file.toFile().exists();
+        } catch (final InvalidPathException ex) {
```

Comment:
@MikhailLipanin AFAIK the `InvalidPathException` is `RuntimeException` and we can remove `catch` clause at all

Reply:
@volodya-lombrozo, Here I use this `try-catch` block to return Boolean value:
- `true` if object found
- `false` if object didn't found or we couldn't check it (throwed an Exception)

Reply:
this logic (`false` if the path of object is invalid) will be used in `ProbeMojo`

Reply:
@MikhailLipanin I still believe you can avoid `InvalidPathException` check. It's quite strange that you have to even check it. Moreover you are using `Exception` for flowcontrol. Please, read https://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#exceptions